Translation/en/Text/Override.txt

```diff
@@ -252,6 +253,8 @@ EA开放=EA Open
 金属性效果=Metal
 
 属性=<size\=18>Stats</size>
+剩余点数:0=<size\=15>Remaining pts: 0</size>

```

Comment:
you don't need this line because the regex below should already cover it

Reply:
I swear my game picks up 2 different colons